woocommerce/woocommerce-android
# [Product discount] Percentage discount picker

### Percentage discount picker
⚠️ Should be reviewed and merged after https://github.com/woocommerce/woocommerce-android/pull/9392 because this branch originates from `9352-product-discounts-full-flow` (pending merge to trunk).



### Description

This PR adds possibility to enter discount amount as a percentage of the item price.

### Testing instructions

1. Create order containing items.
2. Click on the item, and then on product details screen click "+ Add discount"
3. Select a percentage discount type and enter the amount
4. Verify the correct amount is applied to the order

### Images/gif


https://github.com/woocommerce/woocommerce-android/assets/4527432/41774979-d588-4038-978e-e540494da616
